vms/platformvm/blocks/builder/builder_test.go

```diff
@@ -41,16 +45,18 @@ func TestBlockBuilderAddLocalTx(t *testing.T) {
 	env := newEnvironment(t)
 	env.ctx.Lock.Lock()
 	defer func() {
-		require.NoError(shutdownEnvironment(env))
+		env.ctx.Lock.Unlock()
```

Comment:
nit: Unlock follow Shutdown call in prod code. I would keep that order here

Reply:
Hmm I was thinking that Shutdown might try to grab the lock